toqito/channel_ops/choi_to_kraus.py

```diff
@@ -24,7 +24,8 @@ def choi_to_kraus(
     For completely positive maps the output is a single flat list of numpy arrays since the left and
     right Kraus maps are the same.
 
-    This function has been adapted from [Rigetti20]_ and QETLAB package.
+    This function has been adapted from `Rigetti <https://github.com/rigetti/forest-benchmarking>`__ 
+    and QETLAB :cite:`QETLAB_Kraus_Operators`.
```

Comment:
Maybe we just have one reference for QETLAB instead of multiple ones for each function/feature?

Reply:
Fine by me.